packages/sanity/src/core/studio/StudioLayout.tsx

```diff
@@ -84,15 +84,19 @@ export function StudioLayout() {
     const mainTitle = title || startCase(name)
 
     if (activeToolName) {
-      return `${mainTitle} – ${startCase(activeToolName)}`
+      return `${startCase(activeToolName)} | ${mainTitle}`
     }
 
     return mainTitle
   }, [activeToolName, name, title])
 
   useEffect(() => {
+    if (activeToolName === 'content') {
```

Comment:
A potential issue is that tool names can be customized per studio instance, so it's not guaranteed that the desk tool is named `content`. If it's customized, we'll then end up with two code paths fighting to set the document title. Wondering if it would make sense to let tools themselves declare that they controls `document.title`? This will also enable other tools to declare that they take control over updating `document.title` without risking getting into a fight.

I.e. introduce an optional boolean property to the [Tool](https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/core/config/types.ts#L113) interface (e.g. `controlsDocumentTitle` or something), and then replace this check against `tool.name` with a check against this property instead?

(_Ideally_ perhaps it should be a function that we could call from here to get the title, but since it requires so much state I think exposing a flag is better, at least for now)

Reply:
Thanks @bjoerge that makes sense. 
Added `controlsDocumentTitle` property to the tool and set it as true in the Desk tool. 
I agree with you in the ideal behavior but in this case, given it needs to access the panes and resolve them, and also resolve the document if needed, I couldn't make it fit into a function like that. 

commit: https://github.com/sanity-io/sanity/pull/4887/commits/aefe83b63166b6405ee4cd7c3d5cfe0f15d0a9f9